visual_regression_tracker/visualRegressionTracker.py

```diff
@@ -104,14 +104,15 @@ def _http_request(url: str, method: str, data: dict, headers: dict) -> dict:
     request = getattr(requests, method.lower())
     response = request(url, json=data, headers=headers)
     status = response.status_code
+    result = response.json()
 
     if status == 401:
         raise Exception('Unauthorized')
     if status == 403:
         raise Exception('Api key not authenticated')
     if status == 404:
         raise Exception('Project not found')
+    if status >= 400:
+        raise Exception(result)
```

Comment:
Two thoughts:
1. this is populating the exception's message field with a dict. It would be more user friendly to populate it with the message from the response, and keep the rest as a property on the exception.
2. It is getting to the point that is worth having a project specific exception class.

I'm happy to make the changes if you would like.

Reply:
sure, please,  add it

Reply:
I've had a go, but I'm not sure that I've got the exception names quite right. Got any recommendations?

Reply:
I could suggest only to rename `TestFailed` into `TestRunError`

Reply:
I made this change, please, feel free to change it if does not met your expectations

Reply:
👍